regtests/ww3_ufs_weather/input/switch

```diff
@@ -0,0 +1 @@
+F90 NCO NOGRB DIST MPI OMPG OMPH SCRIP SCRIPNC WRST NC4 PR3 UQ FLX0 SEED ST4 STAB0 NL1 BT1 DB1 MLIM FLD2 TR0 BS0 XX0 RWND WNX1 WNT1 CRX1 CRT1 O0 O1 O2 O3 O4 O5 O6 O7 O14 O15 IC0 IS0 REF0
```

Comment:
This is a duplicate (or I'm assuming should be a duplicate) of what is in model/esmf/switch, so maybe this should just be linked?